src/app/api/users/route.js

```diff
@@ -5,7 +5,7 @@ import { NextResponse } from 'next/server';
 export async function GET() {
   try {
     await connectToDataBase();
-    const users = await User.find().populate('preferences');
```

Comment:
Changes the users from array to obj

Reply:
It just fetch one user instead of  all users, exactly  what we need

Reply:
True, and I guess this was not used anywhere before, so does not matter. I receive 500 error, I guess it's related to this I commented below: 
{
error: "Schema hasn't been registered for model "Preference".
Use mongoose.model(name, schema)"
}